### PR TITLE
Add camel-whatsapp wrapper

### PR DESCRIPTION
**Pull request by f2par0**
camel-whatsapp wrapper

### CODE REVIEW

**Review by lopushen — COMMENTED** [review]
Review in progress

File: features/src/main/feature/camel-features.xml [hunk 1]
```diff
@@ -2044,6 +2044,13 @@
         <feature version='${project.version}'>camel-core</feature>
         <bundle>mvn:org.apache.camel.karaf/camel-webhook/${project.version}</bundle>
     </feature>
+    <feature name='camel-whatsapp' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
+        <feature version='${project.version}'>camel-webhook</feature>
+        <feature>jackson</feature>
```
Comment (lopushen) [inline features/src/main/feature/camel-features.xml]: jackson feature contains 2 more dependencies than required
`        <bundle dependency='true'>mvn:com.fasterxml.jackson.jr/jackson-jr-objects/${jackson2-version}</bundle>
        <bundle dependency='true'>mvn:com.fasterxml.jackson.dataformat/jackson-dataformat-cbor/${jackson2-version}</bundle>`
is that ok, or should we define a separate feature for the 3 needed ones?

Comment (essobedo) [reply features/src/main/feature/camel-features.xml]: I agree with you those 2 bundles should not be in the Jackson feature. I think, we need to remove them in the Jackson feature, and add them where it is really needed. Let me fix that.

Comment (f2par0) [reply features/src/main/feature/camel-features.xml]: good catch, thanks

Comment (f2par0) [reply features/src/main/feature/camel-features.xml]: for now, I included only the necessary bundles

Comment (essobedo) [reply features/src/main/feature/camel-features.xml]: No keep it as it was, your feature was good only the Jackson feature had to be fixed

Comment (essobedo) [reply features/src/main/feature/camel-features.xml]: I'm fixing it

Comment (f2par0) [reply features/src/main/feature/camel-features.xml]: ok sorry, did not see your comment before

Comment (essobedo) [reply features/src/main/feature/camel-features.xml]: it's fixed, you can pull the changes

Comment (f2par0) [reply features/src/main/feature/camel-features.xml]: Done

**Review by lopushen — APPROVED** [review]
no conflict ✔️
compilation works ✔️
no unnecessary usage of wrap protocol ✔️
no unnecessary dependencies ✔️
no hardcoded version if it's in the parent pom ✔️
feature is placed in alphabetical order in the file camel-features.xml ✔️
usage of dependent features (netty, jackson, jakarta ...) ✔️
dependent bundles contain dependency="true" ✔️
versions correspond to the dependency tree - jackson is 2.16.2 vs 2.16.1 in the tree, but that's throughout the whole project  ✔️

LGTM

File: features/src/main/feature/camel-features.xml [hunk 2]
```diff
@@ -2106,6 +2106,13 @@
         <feature version='${project.version}'>camel-core</feature>
         <bundle>mvn:org.apache.camel.karaf/camel-webhook/${project.version}</bundle>
     </feature>
+    <feature name='camel-whatsapp' version='${project.version}' start-level='50'>
+        <feature version='${project.version}'>camel-core</feature>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]:
```suggestion
        <feature version='${camel.osgi.version.range}'>camel-core</feature>
```

File: features/src/main/feature/camel-features.xml [hunk 3]
```diff
@@ -2106,6 +2106,13 @@
         <feature version='${project.version}'>camel-core</feature>
         <bundle>mvn:org.apache.camel.karaf/camel-webhook/${project.version}</bundle>
     </feature>
+    <feature name='camel-whatsapp' version='${project.version}' start-level='50'>
+        <feature version='${camel.osgi.version.range}'>camel-core</feature>
+        <feature version='${project.version}'>camel-webhook</feature>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]:
```suggestion
        <feature version='${camel.osgi.version.range}'>camel-webhook</feature>
```

File: features/src/main/feature/camel-features.xml [hunk 4]
```diff
@@ -2106,6 +2106,13 @@
         <feature version='${project.version}'>camel-core</feature>
         <bundle>mvn:org.apache.camel.karaf/camel-webhook/${project.version}</bundle>
     </feature>
+    <feature name='camel-whatsapp' version='${project.version}' start-level='50'>
+        <feature version='${camel.osgi.version.range}'>camel-core</feature>
+        <feature version='${camel.osgi.version.range}'>camel-webhook</feature>
+        <feature>jackson</feature>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]:
```suggestion
        <feature version="[2.16,2.17)">jackson</feature>
```

**Review by essobedo — APPROVED** [review]
I added the version ranges

**Pull request merged**